### PR TITLE
fix(expr): fix graphviz rendering

### PR DESCRIPTION
**Pull request by chloeh13q**
The expression rendering is broken (as shown in the picture below):
broken_graphviz

This PR fixes the expression rendering:
fixed-graphviz

### CODE REVIEW

File: ibis/expr/visualize.py [hunk 1]
```diff
@@ -41,37 +41,52 @@ def get_type(node):
...
     )
 
 
 def get_label(node):
     typename = get_type(node)  # Already an escaped string
+    # If the typename is of the type array<[type]>, e.g. array<int8>,
+    # graphviz will try to render <int8> as an HTML tag and fail
+    if typename.startswith("array<") and typename.endswith(">"):
+        typename = typename.replace("<", "[")
+        typename = typename.replace(">", "]")
```
Comment (cpcloud) [inline ibis/expr/visualize.py:56]: Would `urllib.parse.quote_plus(typename)` also work?

Comment (cpcloud) [reply ibis/expr/visualize.py:56]: Nevermind, there's a different way to do this!

Comment (cpcloud) [reply ibis/expr/visualize.py]: Went ahead and did it in the interest of landing the PR!

**Review by cpcloud — APPROVED** [review]

File: ibis/expr/visualize.py (hunk 1 above)
Comment (chloeh13q) [reply ibis/expr/visualize.py]: Thank you!

**Pull request merged**